FlowCrypt/Functionality/Services/GoogleUserService.swift

```diff
@@ -164,46 +157,26 @@ extension GoogleUserService {
         return GTMAppAuthFetcherAuthorization(fromKeychainForName: Constants.index + email)
     }
 
+    // todo - isn't this call supported by Google client library?
```

Comment:
Google library supports this request - https://github.com/google/google-api-objectivec-client-for-rest/blob/main/Source/GeneratedServices/Oauth2/GTLROauth2Query.m#L46

The only difference is that it uses `oauth2/v2/userinfo` instead of `oauth2/v3/userinfo`, but response looks similar.

Reply:
👍 will file another issue, see if we can use that - for consistency